train.py

```diff
@@ -73,12 +75,23 @@ def generate_subspectrogram(duration_s=config.SUBSPECTROGRAM_DURATION_S, fft_rat
         checkpoint.epoch.assign_add(1)
         checkpoint_manager.save()
 
-    print("======================== evaluation on validation data =========================")
-    # test model on validation set
-    utils.test_model(model, val_spectrograms, val_labels, val_loss, val_accuracy)
+        if epoch % 10 == 0 or epoch == config.NB_EPOCHS - 1:
+            print("======================== evaluation on validation data =========================")
+            # test model on validation set
+            this_val_accuracy, this_val_loss = utils.test_model(
+                model, val_spectrograms, val_labels, val_loss, val_accuracy, epoch=epoch+1)
```

Comment:
Why `epoch+1` ? Everywhere else we use `epoch`, starting at 0.

Reply:
yeah, just seems weird